distsql/distsql.go

```diff
@@ -327,16 +330,6 @@ func SelectDAG(goCtx goctx.Context, ctx context.Context, kvReq *kv.Request, fiel
 
 // Analyze do a analyze request.
 func Analyze(ctx goctx.Context, client kv.Client, kvReq *kv.Request) (SelectResult, error) {
-	var err error
-	defer func() {
-		// Add metrics.
-		if err != nil {
-			queryCounter.WithLabelValues(queryFailed).Inc()
```

Comment:
Why is this removed?

Reply:
It's not accurate on failed or success, the actual error will be returned when we read the result.
And it doesn't provide much useful information.

Reply:
Got it, and please update the description of this PR.